src/community/youtube-channel.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: section
+title: 'Beam YouTube channel'
+section_menu: section-menu/community.html
+permalink: /community/youtube-channel/
+---
+
+# Beam YouTube channel
```

Comment:
Noting a followup (not in this PR - let's get this in and iterate): I would love for this page to be a little more user focused, like "Apache Beam has a community-built YouTube channel where videos about Beam are collected. It is a great resource to learn about using and contributing to the project." It would replace the "goal" section with a "why _you the user_ care" section.

And then below that talk about the process of adding a video just like you do now.